spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AdminUser, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
What is this doing?

Reply:
This file was added by active admin, I've found other examples online and they all seem to contain code relevant for creating a more accurate login page (making sure the username contains a valid email address, rejecting those that are invalid, rejecting two accounts with the same email address, etc.)